src/helpers.js

```diff
@@ -5,13 +5,18 @@ const TEXT_NODE = 3
 
 // Currently this fn only supports jest timers, but it could support other test runners in the future.
 function runWithRealTimers(callback) {
-  // istanbul ignore else
-  if (typeof jest !== 'undefined') {
-    return runWithJestRealTimers(callback).callbackReturnValue
-  }
+  return hasJestTimers()
+    ? runWithJestRealTimers(callback).callbackReturnValue
+    : // istanbul ignore next
+      callback()
+}
 
-  // istanbul ignore next
-  return callback()
+function hasJestTimers() {
+  return (
+    jest !== undefined &&
```

Comment:
```suggestion
    typeof jest !== 'undefined' &&
```
To guard against `ReferenceError`